src/main/java/Ui.java

```diff
@@ -10,7 +10,7 @@ public Ui() {
         this.parser = new Parser();
     }
 
-    public static String LINE = "___________________________________________________";
```

Comment:
Since the line is not to share with other class, maybe change it to private instead?